src/win32_surface.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+#![cfg(windows)]
```

Comment:
No need to put it here since you import this file only if we're building on windows.